src/coreclr/vm/virtualcallstub.cpp

```diff
@@ -3818,8 +3770,8 @@ BOOL VirtualCallStubManagerManager::CheckIsStub_Internal(
     WRAPPER_NO_CONTRACT;
     SUPPORTS_DAC;
 
-    VirtualCallStubManager *pMgr = FindVirtualCallStubManager(stubStartAddress);
-    return (pMgr != NULL);
+    // Forwarded to from RangeSectionStubManager
+    return FALSE;
```

Comment:
Why return false on this case? Is it because this was supporting cross domain tracing?